.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Description
+
+<!-- Describe the bug this PR fixes or the feature it adds. Link to any related issues and PRs -->
+
+#### Breaking change?
+
+<!-- If applicable, list the APIs/functionality which this PR breaks -->
+
+### Example
+
+<!-- If applicable, add an example on how this improves the application -->
+
+---
+
+### Checklist
+
+- [ ] All tests pass
```

Comment:
The CI should check for that so I'm not sure it's useful to have it as a checklist item @smcclellan